### PR TITLE
BSM-88: Implement alarm table polling functionality

### PR DESCRIPTION
**Pull request by mvrueden**
* JIRA: http://issues.opennms.org/browse/BSM-88
* Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS388

Please note that with this PR issue http://issues.opennms.org/browse/BSM-79 is also fixed.

### CODE REVIEW

File: opennms-base-assembly/src/main/filtered/etc/create.sql [hunk 1]
```diff
@@ -2666,3 +2666,13 @@ CREATE TABLE bsm_service_ifservices (
   REFERENCES bsm_service (id) ON DELETE CASCADE
 );
 
+CREATE TABLE bsm_service_children (
```
Comment (j-white) [inline opennms-base-assembly/src/main/filtered/etc/create.sql]: Why is this necessary?

Comment (mvrueden) [inline opennms-base-assembly/src/main/filtered/etc/create.sql]: Actually we don't.
